docs/docs/guides/integrate/login-ui/passkey.mdx

```diff
@@ -0,0 +1,308 @@
+
+### Request
+```bash
+curl --request POST \
+  --url https://$ZITADEL_DOMAIN/v2alpha/users/$USER_ID/passkeys/registration_link \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer '"$TOKEN"''\
+  --header 'Content-Type: application/json' \
+  --data '{
+   "sendLink": {
+      "urlTemplate": "https://example.com/passkey/register?userID={{.UserID}}&orgID={{.OrgID}}&codeID={{.CodeID}}&code={{.Code}}"
+   },
+   "returnCode": {}
+}'
+```
+
+## Response in case of return code
+```bash
+{
+	"details": {
+		"sequence": "632",
+		"changeDate": "2023-06-28T08:09:51.257699Z",
+		"resourceOwner": "163840776835432705"
+	},
+	"code": {
+		"id": "220526087715684609",
+		"code": "2KEpIeQGSBBd"
+	}
+}
+```
+
+## Start Passkey Registration
+
+When starting the passkey registration you can optionally send the code from the step before, if you have sent the code to the user.
+
+By specifying the authenticator you can choose if the passkey should be cross paltform or not. Per default everything is allowed:
+- PASSKEY_AUTHENTICATOR_UNSPECIFIED
+- PASSKEY_AUTHENTICATOR_PLATFORM
+- PASSKEY_AUTHENTICATOR_CROSS_PLATFORM
+
+In the response you will get the public key credential options, that will be obtained by the browser to use the physical key to obtain a signed challenge.
+
+[Start Passkey Registration Documentation](/apis/resources/user_service/user-service-register-passkey)
+
+### Request
+
+```bash
+curl --request POST \
+  --url https://$ZITADEL_DOMAIN/v2alpha/users/$USER_ID/passkeys \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer '"$TOKEN"''\
+  --header 'Content-Type: application/json' \
+  --data '{
+	"code": {
+		"id": "220526087715684609",
+		"code": "2KEpIeQGSBBd"
+	},
+  "authenticator": "PASSKEY_AUTHENTICATOR_UNSPECIFIED"
+}'
+```
+
+### Response
+
+```bash
+{
+	"details": {
+		"sequence": "633",
+		"changeDate": "2023-06-28T08:10:26.725981Z",
+		"resourceOwner": "163840776835432705"
+	},
+	"passkeyId": "220526147258024193",
+	"publicKeyCredentialCreationOptions": {
+		"publicKey": {
+			"attestation": "none",
+			"authenticatorSelection": {
+				"userVerification": "required"
+			},
+			"challenge": "JM1uLbVQR2xZJ210DA7E-3j0Cd9rHKUSmc8NyIJBtAY",
+			"pubKeyCredParams": [
+				{
+					"alg": -7,
+					"type": "public-key"
+				},
+				{
+					"alg": -35,
+					"type": "public-key"
+				},
+				{
+					"alg": -36,
+					"type": "public-key"
+				},
+				{
+					"alg": -257,
+					"type": "public-key"
+				},
+				{
+					"alg": -258,
+					"type": "public-key"
+				},
+				{
+					"alg": -259,
+					"type": "public-key"
+				},
+				{
+					"alg": -37,
+					"type": "public-key"
+				},
+				{
+					"alg": -38,
+					"type": "public-key"
+				},
+				{
+					"alg": -39,
+					"type": "public-key"
+				},
+				{
+					"alg": -8,
+					"type": "public-key"
+				}
+			],
+			"rp": {
+				"id": "fabi-t7rlmd.zitadel.app",
```

Comment:
Do we want to keep this domain here?

Reply:
No I change it to make it clear it is an example